cellprofiler/modules/morph.py

```diff
@@ -238,14 +238,28 @@
 
     - Binary
 
-|
-
-============ ============
-Supports 2D? Supports 3D?
-============ ============
-YES          NO
-============ ============
-
+Upgrading:
+~~~~~~~~~~
+
+The following **Morph** operations have been extracted to separate modules in CellProfiler 3.0.
+Use the table below to update your pipeline to use the corresponding module and, where appropriate,
+setting and value.
+
+=================  ======================  ===========  =======================
+Morph operation    Module                  Setting      Value
+=================  ======================  ===========  =======================
+bothat             TopHatTransform*        *Operation*  Black top-hat transform
+close              Closing
+dilate             Dilation
```

Comment:
I'm currently working on a `DilateObjects` module. Since it seems like these have been converted to plugins, should that module go to CellProfiler-plugins as well?

Reply:
I think DilateObjects makes sense in CellProfiler-plugins.